src/main.rs

```diff
@@ -29,9 +32,11 @@ fn main() {
     let opts = Opts::parse();
 
     let file = opts.file.unwrap_or_else(|| String::from("/dev/stdin"));
+    // aardvark config directory must be supplied by parent or it defaults to /tmp/aardvark
+    let config = opts.config.unwrap_or_else(|| String::from("/tmp/aardvark"));
```

Comment:
Why do we have a default? Just keep it as an option and when a network has dns enabled and the config dir is not set return an error.

Reply:
I just added it till we add support in upstream `podman` for `config` otherwise returning `error` will break `podman`. So this is a placeholder we can get rid of it later.

Reply:
ok